test/behaviour/concept/thing/entity/BUILD

```diff
@@ -19,7 +19,7 @@
 
```

Comment:
The rest of these test files are just identical changes to the above - they are not interesting